crates/bevy_camera_controller/src/free_cam.rs

```diff
@@ -124,7 +146,11 @@ Freecam Controls:
     }
 }
 
-fn run_camera_controller(
+/// This system is typically added via the [`FreeCamPlugin`].
+///
+/// Reads inputs and then moves the camera entity according
+/// to the settings given in [`FreeCam`].
+pub fn run_freecam_controller(
```

Comment:
NB: Another piece of Bevy that is a little bit hamstrung by not having BEI upstream. Could we open an issue to update the controls to BEI to allow the user to tweak them once we have it upstreamed?